reference/onenote/notebook.md

```diff
@@ -9,6 +9,7 @@ Represents a OneNote notebook. Notebooks contain section groups and sections.
 
 | Property	   | Type	|Description|Feedback|
 |:---------------|:--------|:----------|:-------|
+|baseUrl|string|The base site url for the notebook, if it is in a SharePoint site (it will be null for OneDrive notebooks) - this is to be used for interacting with the OneNote REST API, according to our [OneNote API documentation](https://blogs.msdn.microsoft.com/onenotedev/2015/06/11/and-sharepoint-makes-three/) to fetch the SiteCollectionId and SiteId (using FromUrl). Read only.|[Go](https://github.com/OfficeDev/office-js-docs/issues/new?title=OneNote-notebook-baseUrl)|
```

Comment:
Remove the hyphen and create a new sentence as follows: "Use this property to interact with the OneNote REST API to fetch the **SiteCollectionId** and **SiteId** (using **FromUrl**). Read-only. For details, see the [OneNote development blog](https://blogs.msdn.microsoft.com/onenotedev/2015/06/11/and-sharepoint-makes-three/).

Note that programming elements should have bold formatting applied.